webpack.config.js

```diff
@@ -53,6 +53,7 @@ const entryPointNames = [
 	'i18n',
 	'utils',
 	'data',
+	'edit-post',
```

Comment:
To be able to export this as `wp.editPost` we should consider mapping packages using [Lodash's `_.camelCase`](https://lodash.com/docs/4.17.4#camelCase).

Reply:
Might not be possible to configure webpack to do so https://webpack.js.org/configuration/output/#output-library